src/tms_service.ts

```diff
@@ -45,6 +51,21 @@ type EmsRasterStyle = {
 };
 
 export class TMSService extends AbstractEmsService {
+  /*
+  List of supported languages with labels and OMT code
+  */
+  public static SupportedLanguages = {
+    en: { label: 'English', omtCode: 'en' },
+    'ch-CN': { label: 'Chinese', omtCode: 'zh' },
+    'ja-JP': { label: 'Japanese', omtCode: 'ja' },
+    'fr-FR': { label: 'French', omtCode: 'fr' },
+    es: { label: 'Spanish', omtCode: 'es' },
+    ar: { label: 'Arabic', omtCode: 'ar' },
+    'hi-IN': { label: 'Hindi', omtCode: 'hi' },
+    'ru-RU': { label: 'Russian', omtCode: 'ru' },
+    'pt-PT': { label: 'Portuguese', omtCode: 'pt' },
+  };
+
```

Comment:
Can we make this an array of objects instead? Something like

```
[
  { key: 'en', label: 'English', omtCode: 'en' },
  { key: 'ch-CN', label: 'Chinese', omtCode: 'zh' },
  ...
]
```

Reply:
yep, this makes the code much easier to read and work with, thanks!